packages/ethereum-contracts/contracts/superfluid/Superfluid.sol

```diff
@@ -842,7 +844,9 @@ contract Superfluid is
                 _callAppAction(
                     msgSender,
                     ISuperApp(operations[i].target),
+                    valueForwarded ? 0 : msg.value,
```

Comment:
The assumption is that this will revert when the given `SUPERFLUID_CALL_APP_ACTION` it's passed to is not `payable`.

Reply:
I checked again with a dedicated test. Does actually not fail, it's really just a compiler level construct. There must have been a misleading red herring yesterday when I checked.
Thinking about it, that makes sense, because otherwise the introduction of payable in Solidity would have been a breaking change.

Reply:
yes, I added a test case for that.